testsuite/utils/src/main/resources/META-INF/keycloak-server.json

```diff
@@ -17,10 +17,10 @@
         },
         "map": {
             "storage-admin-events": {
-                "provider": "${keycloak.adminEventsStore.map.storage.provider:concurrenthashmap}"
+                "provider": "${keycloak.adminEventsStore.map.storage.provider,keycloak.mapStorage.provider.default:concurrenthashmap}"
```

Comment:
Should this changes go into `testsuite/integration-arquillian/tests/base/src/test/resources/META-INF/keycloak-server.json` as well?

Reply:
In the arquillian testsuite, there are `map-storage-*` profiles which define all the necessary properties for map storage. Here it is only for convenience usage from CLI. So while I agree it would be nice, I consider this a follow-up task if the need arises.